hub/api/tests/test_rechunk.py

```diff
@@ -69,15 +70,16 @@ def test_rechunk_2(local_ds):
 
 
 def test_rechunk_3():
-    NUM_TEST_SAMPLES = 100
```

Comment:
let's use local_ds as an argument to test_rechunk_3 and use that instead of ds = hub.empty, no need to use an isolated_filesystem here

Reply:
Will do! Before this PR, `test_rechunk_3` used to run while `test_rechunk_1` and `test_rechunk_2` was skipped while running `pytest .`. Now, while using the `local_ds` argument, `test_rechunk_3` also gets skipped along with `test_rechunk_1` and `test_rechunk_2`. Just confirming if this is intended behaviour! @AbhinavTuli

Reply:
yup that is indeed the intended behaviour, pytest . should only run memory tests and not local storage ones, `pytest --local . ` should run all 3